django_project/changes/views/version.py

```diff
@@ -85,6 +92,20 @@ def get_context_data(self, **kwargs):
         if project_slug:
             context['the_project'] = Project.objects.get(slug=project_slug)
             context['project'] = context['the_project']
+
+
+        # lets check for specific user permissions here.
+        if self.request.user.is_authenticated:
+            context['user_can_view'] = True
+
+        if self.request.user.is_staff:
```

Comment:
are you sure that all staff can delete and edit?

Reply:
Yes @cchristelis that was the permission check ealrier before I replaced with the context varible.

Reply:
Updated @cchristelis